pkg/sdk/symbols/info/info.go

```diff
@@ -105,7 +107,27 @@ func plugin_get_required_api_version() *C.char {
 	return (*C.char)(pRequiredAPIVersion.CharPtr())
 }
 
+func string_split(version string) (string, string, string) {
+	nums := strings.Split(version, ".")
+	if len(nums) != 3 {
+		panic("Incorrect format. Expected: Semantic Versioning: X.Y.Z")
+	}
+	return nums[0], nums[1], nums[2]
+}
+
 func SetRequiredAPIVersion(apiVer string) {
+	requiredMajor, requiredMinor, requiredPatch := string_split(apiVer)
+	pluginMajor, pluginMinor, pluginPatch := string_split(C.GoString(plugin_get_required_api_version()))
```

Comment:
```suggestion
	pluginMajor, pluginMinor, pluginPatch := string_split(C.GoString(get_default_required_api_version()))
```
get_default_required_api_version() returns the actual default version of the plugin API definitions underneath.